server/controller/photos.js

```diff
@@ -12,8 +13,12 @@ const photos = (req, res) => {
     if (error) {
       throw error;
     }
-    console.log(results);
-    res.status(200).json(results.rows);
+    const queryData = results.rows;
+
+    const responseData = formatData(queryData, propertyId);
+    console.log(responseData);
+
+    res.status(200).json(responseData);
```

Comment:
same as Justin, do you want to add a catch here to send like a 404 to client if there is no data to send?